Makefile

```diff
@@ -0,0 +1,176 @@
+######################################################
+#        SYSTEM MAKE FILE                            #
+######################################################
+# Use this to allow quick/easy modifications and/or  #
+# updates to the software. Requires make to be       #
+# installed.                                         #
+#                                                    #
+# Note: This assumes you already have PHP setup and  #
+#   properly configured in your path environment     #
+######################################################
+# Windows                                            #
+#                                                    #
+# http://gnuwin32.sourceforge.net/downlinks/make.php #
+######################################################
+# OS X / Linux                                       #
+#                                                    #
+# Likely Already Installed. Otherwise use the os's   #
+# package manager (IE: apt-get, brew, macports, etc) #
+######################################################
```

Comment:
Could you add standard Phalcon's license note at the top

Reply:
Tracking: Done